internal/handler/handler.go

```diff
@@ -290,11 +297,11 @@ func (h *Handler) Collect(ch chan<- prometheus.Metric) {
 
 // cleanupAllCappedCollections drops the given percent of documents from all capped collections.
 func (h *Handler) cleanupAllCappedCollections(ctx context.Context) error {
-	h.L.Debug("cleanupAllCappedCollections: started", zap.Uint8("percentage", h.CappedCleanupPercentage))
+	h.L.DebugContext(ctx, "cleanupAllCappedCollections: started", slog.Any("percentage", h.CappedCleanupPercentage))
```

Comment:
In general, we should try to avoid using `slog.Any` – we don't know what types are really supported by handlers. In that case, we should convert to an integer type

Reply:
Sure, I think only time we use `slog.Any` is reserved for logging `struct`, but even then you are correct about not knowing how handlers handle types.

Reply:
I've left the usage of `slog.Any` for `struct` type and `any` type. For `struct` type it's possible to implement `slog.LogValuer` interface, if we would like. That should be respected by handlers.

Reply:
In fact `struct` type should always implement `slog.LogValuer`